qiskit_nature/results/utils/protein_plotter.py

```diff
@@ -0,0 +1,146 @@
+
+        self._fig = plt.figure()
+        self._ax_graph = self._fig.add_subplot(projection="3d")
+
+    def _draw_main_chain(self):
+        """
+        Draws the main chain.
+
+        """
+        for i, main_aminoacid in enumerate(self._shape_gen._main_chain_aminoacid_list):
+            self._ax_graph.text(
+                self._x_main[i],
+                self._y_main[i],
+                self._z_main[i],
+                main_aminoacid,
+                size=10,
+                zorder=10,
+                color="k",
+            )
+
+        self._ax_graph.plot3D(self._x_main, self._y_main, self._z_main)
+        return self._ax_graph.scatter3D(
+            self._x_main, self._y_main, self._z_main, s=500, label="Main Chain"
+        )
+
+    def _draw_side_chains(self):
+        """
+        Draws the side chain.
+        """
+        side_positions = self._shape_gen.side_positions
+        side_aminoacids = self._shape_gen._main_chain_aminoacid_list
+        for i, side_chain in enumerate(side_positions):
+            if side_chain is not None:
+                x_side, y_side, z_side = side_chain
+                side_scatter = self._ax_graph.scatter3D(
+                    x_side, y_side, z_side, s=600, c="green", label="Side Chain"
+                )
+                self._ax_graph.plot3D(
+                    [self._x_main[i], x_side],
+                    [self._y_main[i], y_side],
+                    [self._z_main[i], z_side],
+                    c="green",
+                )
+                self._ax_graph.text(
+                    x_side,
+                    y_side,
+                    z_side,
+                    side_aminoacids[i],
+                    size=10,
+                    zorder=10,
+                    color="k",
+                )
+        return side_scatter
+
+    def _format_graph(
+        self, title: str, ticks: bool, grid: bool, main_scatter: plt.Axes, side_scatter: plt.Axes
+    ):
+        """
+        Formats the plot.
+        Args:
+            title: The title of the plot.
+            ticks: Boolean for showing ticks in the graphic.
+            grid: Boolean for showing the grid in the graphic.
+            main_scatter: Scattering object that we will use for the legend.
+            side_scatter: Scattering object that we will use for the legend.
+        """
+
+        self._ax_graph.set_box_aspect([1, 1, 1])
+
+        self._ax_graph.grid(grid)
+
+        if not ticks:
+            self._ax_graph.set_xticks([])
+            self._ax_graph.set_yticks([])
+            self._ax_graph.set_zticks([])
+
+        self._ax_graph.set_xlabel("x")
+        self._ax_graph.set_ylabel("y")
+        self._ax_graph.set_zlabel("z")
+
+        self._fig.legend(handles=[main_scatter, side_scatter], labelspacing=2, markerscale=0.5)
+        self._ax_graph.set_title(title)
+
+    def plot(
+        self, title: str = "Protein Structure", ticks: bool = False, grid: bool = False
+    ) -> plt.figure:
```

Comment:
Are we happy to have this called `plot` would `get_plot` or `get_figure` be more descriptive of what it does? In matplot when I do `plot` it renders it too right. I will note that Optimization applications have a draw() method - that I think often goes to networkx draw.

Let me make the comment here while I think about it. Are we going to update the protein folding notebook to include this new interpret capability and show this figure there as well etc.